lib/elixir/lib/string.ex

```diff
@@ -988,6 +988,8 @@ defmodule String do
   @doc """
   Returns all codepoints in the string.
 
+  For details about codepoints and graphemes, [see the module documentation](#moduledoc).
```

Comment:
"See the `String` module documentation". We should not rely on HTML markup (for example, it doesn't make sense on IEx's help).